go/vt/vttablet/tabletmanager/vreplication/flags.go

```diff
@@ -0,0 +1,69 @@
+package vreplication
```

Comment:
Most of this is moved out of `vreplicator.go`. See comments below identifying the parts that are new.